dicom_standard/extract_ciod_func_group_macro_tables.py

```diff
@@ -22,18 +22,18 @@
 )
 
 CHAPTER_ID = 'chapter_A'
-# Standard workaround: Include optional "s" at end of "Functional Group" to catch Table A.32.9-2
-# http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.32.9.3.4.html#table_A.32.9-2
-TABLE_SUFFIX = re.compile(".*Functional Groups? Macros$")
+TABLE_SUFFIX = re.compile(".*Functional Group Macros$")
 COLUMN_TITLES = ['macro', 'section', 'usage']
 
 
-# Standard workaround: Add missing "Image" to title of Table A.52.4.3-1
-# http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.52.4.3.html#table_A.52.4.3-1
 def clean_macro_table_name(table_name: str) -> str:
     clean_name = pl.clean_table_name(table_name)
-    if clean_name == 'Ophthalmic Tomography':
-        clean_name = 'Ophthalmic Tomography Image'
+    # if clean_name == 'Ophthalmic Tomography':
```

Comment:
Dead code?

Reply:
Oh, nice catch. I had initially intended on leaving fixed workarounds as comments but as you saw, I ended up just adding a section to the README.